app/views/groups/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="col-md-6 mx-auto">
+    <div class="categories">
+        <%= render @groups %>
+    </div>
+
+    <%= link_to "Add New Category", new_group_path,  class: " btn btn-default  bg-gradient mb-3 w-100 btn-lg mt-4" %>
+</div>
```

Comment:
- [ ] [OPTIONAL] Great job done @mrprotocoll 👍🏽 . However, I would like to suggest that you should ensure that the `Add New Category` button stays at the button of the page before a user adds a category. This will enable a better user interface and experience (UI/UX).
- [ ] [OPTIONAL] Kindly ensure you do the same for the `Save` button in the Transaction/Purchase page 🥇 

### Your Application on Sign-in
<img width="416" alt="Screenshot 2023-06-03 at 11 12 23" src="https://github.com/microverse-journey/mp-budget/assets/58692590/52da97bf-bf61-4bfc-a6c4-8b23d3613527">